packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -75,6 +76,9 @@ const Trade = observer(() => {
     const [category, setCategory] = React.useState<string>();
     const [subcategory, setSubcategory] = React.useState<string>();
     const [swipe_index, setSwipeIndex] = React.useState<number | undefined>(0);
+    const [open_launch_modal, setOpenLaunchModal] = React.useState<boolean>(true);
+
+    const charts_ref = React.useRef() as React.MutableRefObject<HTMLDivElement>;
```

Comment:
we can get rid of it like it's done on master, `charts_ref` is **not** used by `SmartChart`

Reply:
done , thanks